### DISCUSSION TITLE
v0.27.4

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/UnblockNeteaseMusic#1231] [Announcements]
## TL; DR;
- Fixed kuwo provider
- Fixed dotenv support
- Adapted new NCM API-s

## Breaking changes
- Removed broken yt-download provider

## Fixes
- [feat(hook): add new hook for 163jiasu CDN host](https://github.com/UnblockNeteaseMusic/server/commit/f71cdd1e6c8aa006cdce1be1bbdf3221eaab848a)
- [fix: rework kuwo search api and music upload api](https://github.com/UnblockNeteaseMusic/server/commit/b04c9887965890fc43bfd787d88f2f4851e16eee)
- [fix(dotenv): fix parse value with multiple `=`](https://github.com/UnblockNeteaseMusic/server/commit/bd1f348e82d01015bed7cd13599b0c22478756fc)

## Refactor
- [refactor(provider): remove yt-download sourc](https://github.com/UnblockNeteaseMusic/server/commit/37481b024b747ed85ef93b65a38ceca2004d0e94)

## Chore
- Update dependencies to the latest

## New Contributors
* @LittleRey made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1206
* @Inkarro made their first contribution in https://github.com/UnblockNeteaseMusic/server/pull/1207

**Full Changelog**: https://github.com/UnblockNeteaseMusic/server/compare/v0.27.3...v0.27.4

<hr /><em>This discussion was created from the release <a href='https://github.com/UnblockNeteaseMusic/server/releases/tag/v0.27.4'>v0.27.4</a>.</em>
